Vienna/Sources/Main window/WebKitArticleConverter.swift

```diff
@@ -89,25 +90,38 @@ class WebKitArticleConverter: ArticleConverter {
         }
     }
 
-    func prepareArticleDisplay(_ articles: [Article]) -> (URL) {
+    func prepareArticleDisplay(_ articles: [Article]) -> (htmlPath: URL, accessPath: URL) {
+        prepareArticleDisplay(articles, articleFileName: "article.html")
```

Comment:
Did you really intend to do this ? all the files will end with the name "article.html"